src/main/java/no/nav/foreldrepenger/oversikt/innhenting/BehandlingHendelseHåndterer.java

```diff
@@ -52,13 +52,25 @@ void handleMessage(String topic, String key, String payload) {
             if (hendelse.getHendelse().equals(Hendelse.MIGRERING)) {
                 hentSakMedEnGang(hendelse);
             } else if (!IGNORE.contains(hendelse.getHendelse())) {
-                lagreHentSakTask(hendelse.getHendelseUuid(), new Saksnummer(hendelse.getSaksnummer()));
+                if (erNyOppdatering(hendelse)) {
+                    lagreHentSakTask(hendelse.getHendelseUuid(), new Saksnummer(hendelse.getSaksnummer()));
+                } else {
+                    LOG.info("Lager ikke task for å hente sak. Allerede oppdatert. {}", hendelse.getSaksnummer());
+                }
             }
         } catch (Exception e) {
             LOG.warn("Feilet ved håndtering av hendelse. Ignorerer {}", key, e);
         }
     }
 
+    private boolean erNyOppdatering(BehandlingHendelseV1 hendelse) {
+        if (hendelse.getTidspunkt() == null) {
+            return true;
+        }
+        var eksisterende = sakRepository.hent(new Saksnummer(hendelse.getSaksnummer()));
+        return eksisterende.map(sak -> sak.oppdatertTidspunkt().isBefore(hendelse.getTidspunkt())).orElse(true);
```

Comment:
Hvor kommer oppdatertTidspunkt fra? Hvilket tidspunkt tenker du bruke?

Reply:
Det settes rett før fpoversikt lagrer oppdatert versjon av saken i databasen. Altså det er fpoversikt sitt eget tidspunkt for oppdatering, ikke fpsak sitt tidspunkt for endring på saken